pydm/widgets/shell_command.py

```diff
@@ -228,7 +256,12 @@ def execute_command(self, command):
             args = shlex.split(command, posix='win' not in sys.platform)
             try:
                 logger.debug("Launching process: %s", repr(args))
-                self.process = subprocess.Popen(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+                if self._redirect_output:
```

Comment:
This is the easiest way to deal with it, but I could do something like:

```
 out, err = self.process.communicate()
 sys.stdout.write(out) # and will have to deal here with Python 3 vs Python 2
```
or  something like this:
```
 for l in iter(process.stdout.readline, ''):  # and replace '' with b'' for Python 3
        sys.stdout.write(l)
```

although I don't think we need to process the output if we don't need too, what do you think @hhslepicka